src/main/java/tech/gruppone/stalker/server/model/Connection.java

```diff
@@ -0,0 +1,20 @@
+package tech.gruppone.stalker.server.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.springframework.data.relational.core.mapping.Column;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Connection {
+
+    @NonNull
+    @Column("organizationId")
+    Long organizationId;
+
+}
```

Comment:
dovrebbe andare in model.db ed essere una "traduzione" fedele di una riga di Connection nel db